### PR TITLE
move resource names

### PR DESCRIPTION
**Pull request by pongad**

### CODE REVIEW

File: google-api-grpc/pom.xml [hunk 1]
```diff
@@ -496,6 +496,11 @@
         <artifactId>grpc-google-cloud-websecurityscanner-v1alpha</artifactId>
         <version>0.12.1-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-websecurityscanner-v1alpha:current} -->
       </dependency>
+      <dependency>
+        <groupId>com.google.api.grpc</groupId>
+        <artifactId>resource-name-common</artifactId>
```
Comment (garrettjonesgoogle) [inline google-api-grpc/pom.xml]: Maybe we should use `google-cloud-resource-name-common`

File: versions.txt [hunk 2]
```diff
@@ -79,8 +79,8 @@ proto-google-cloud-vision-v1:1.11.0:1.11.1-SNAPSHOT
 proto-google-cloud-vision-v1p1beta1:0.12.0:0.12.1-SNAPSHOT
 proto-google-cloud-vision-v1p2beta1:1.11.0:1.11.1-SNAPSHOT
 proto-google-cloud-cloudiot-v1:0.12.0:0.12.1-SNAPSHOT
-proto-google-cloud-redis-v1beta1:10.12.0:0.12.1-SNAPSHOT
-proto-google-cloud-websecurityscanner-v1alpha:10.12.0:0.12.1-SNAPSHOT
+proto-google-cloud-redis-v1beta1:0.12.0:0.12.1-SNAPSHOT
+proto-google-cloud-websecurityscanner-v1alpha:0.12.0:0.12.1-SNAPSHOT
```
Comment (garrettjonesgoogle) [inline versions.txt]: Could you fix these in a separate PR? this is very hidden as it is.

**Comment by pongad** [comment]
@garrettjonesgoogle PTAL

File: google-api-grpc/google-cloud-resource-name-common/pom.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>google-cloud-resource-name-common</artifactId>
+  <version>0.12.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-resource-name-common:current} -->
+  <name>proto-google-cloud-websecurityscanner-v1alpha</name>
```
Comment (garrettjonesgoogle) [inline google-api-grpc/google-cloud-resource-name-common/pom.xml]: Wrong name here

File: google-api-grpc/google-cloud-resource-name-common/pom.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  <description>PROTO library for proto-google-cloud-websecurityscanner-v1alpha</description>
```
Comment (garrettjonesgoogle) [inline google-api-grpc/google-cloud-resource-name-common/pom.xml]: Remove the "PROTO" word from the description

File: google-api-grpc/google-cloud-resource-name-common/pom.xml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  <parent>
+    <groupId>com.google.api.grpc</groupId>
+    <artifactId>google-api-grpc</artifactId>
+    <version>0.12.1-SNAPSHOT</version><!-- {x-version-update:google-api-grpc:current} -->
+  </parent>
+  <dependencies>
+    <dependency>
+      <groupId>com.google.api</groupId>
+      <artifactId>api-common</artifactId>
+      <scope>compile</scope>
```
Comment (garrettjonesgoogle) [inline google-api-grpc/google-cloud-resource-name-common/pom.xml]: Isn't this scope implied?

File: google-api-grpc/google-cloud-resource-name-common/pom.xml (hunk 3 above)
Comment (pongad) [reply google-api-grpc/google-cloud-resource-name-common/pom.xml]: Welp, that's embarassing

File: google-api-grpc/google-cloud-resource-name-common/pom.xml (hunk 4 above)
Comment (pongad) [reply google-api-grpc/google-cloud-resource-name-common/pom.xml]: Done

File: google-api-grpc/google-cloud-resource-name-common/pom.xml (hunk 5 above)
Comment (pongad) [reply google-api-grpc/google-cloud-resource-name-common/pom.xml]: You're right, that's the default

**Pull request merged**